sdk/include/ndk/mmtypes.h

```diff
@@ -100,11 +101,13 @@ extern "C" {
 #define SEC_NOCACHE                                         0x10000000
 #define SEC_WRITECOMBINE                                    0x40000000
 #define SEC_LARGE_PAGES                                     0x80000000
-#else
+
+#else // NTOS_MODE_USER
+
 #define SEC_BASED                                           0x200000
 
 //
-// Section Inherit Flags for NtCreateSection
+// Section Inherit Flags for NtMapViewOfSection and related
```

Comment:
"and related": are there other exported functions that also use the SECTION_INHERIT?

Reply:
No, removed.